src/integrationTest/java/uk/gov/hmcts/reform/blobrouter/data/ReconciliationReportRepositoryTest.java

```diff
@@ -207,6 +219,35 @@ void should_find_only_latest_report_when_conditions_are_met() {
             });
     }
 
+    @Test
+    void should_find_latest_report_by_datetime_even_if_was_generated_on_another_day() {
+        // given
+        // original report was generated yesterday
+        TestClockProvider.stoppedInstant = ZonedDateTime.now(TimeZones.EUROPE_LONDON_ZONE_ID).minusDays(1)
+            .toInstant();
+        saveNewReportAndGetId("{}", "{ \"x\": 983 }", now().minusDays(1));
+
+        // newest supplier statement by date new report was regenerated today
+        TestClockProvider.stoppedInstant = ZonedDateTime.now(TimeZones.EUROPE_LONDON_ZONE_ID).toInstant();
+        var newReport = saveNewReportAndGetId("{}", "{ \"x\": 666 }",
+                                              now().minusDays(1)
+        );
```

Comment:
this is extremely weird formatting (for this project) ;)
Maybe it will fit into single line?